.pre-commit-config.yaml

```diff
@@ -154,6 +154,11 @@ repos:
             pandas/tests/io/excel/test_writers\.py|
             pandas/tests/io/pytables/common\.py|
             pandas/tests/io/pytables/test_store\.py$
+    -   id: check-no-pd-class
+        name: check for pd class.
+        types: [python]
+        entry: pd.testing.assert_series_equal
```

Comment:
this needs to be pd.testing.* (not exatcly sure how to do the wildcard here). cc @MarcoGorelli

Reply:
ah I see, thanks for the feed back. If my knowledge is correct, a wild card is unnecessary. I'll update this.